files/js-cookie/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "js-cookie",
+  "repo": "js-cookie/js-cookie",
+  "files": {
+    "include": ["src/js.cookie.js"]
```

Comment:
Neither, only the source. The client is responsible to minify the main file using their specific build routine.

For those using it in a simple website, they use CDN or online tools.